Bugsnag/Helpers/BSGRunContext.h

```diff
@@ -0,0 +1,24 @@
+//
+//  BSGRunContext.h
+//  Bugsnag
+//
+//  Copyright © 2022 Bugsnag Inc. All rights reserved.
+//
+
+#define BSGRUNCONTEXT_VERSION 1
```

Comment:
Can we put a comment here that discusses how/when this gets updated?

Reply:
I've added some comments here, although it's made me question whether we really need a version field at this stage - checking the struct size is enough...

Reply:
I think we should keep it to stay on the safe side, in case a struct change results in a total size we've already used in the past.

Also, a "struct data is valid" flag would probably solve the other issue I mentioned. Set it to true once struct initialization is complete. Then we have three checks to make: Does the version match, does the size match, is the "valid" flag true.

Reply:
Sure, it doesn't hurt to check the version.

That said, I think it makes sense to follow a rule of never removing a field from the struct.

The `structVersion` field will also act as a "struct data is valid" flag.

Reply:
Sorry for the ninja edit :)